### PR TITLE
Support Rotation

### PR DESCRIPTION
**Pull request by coxley**
This change adds rotation support to the `xrandr` plugin.

### CODE REVIEW

File: py3status/modules/xrandr.py [hunk 1]
```diff
@@ -248,15 +250,16 @@ def _apply(self, force=False):
             #
             if output in combination:
                 pos = getattr(self, '{}_pos'.format(output), '0x0')
+                rotation = getattr(self, '{}_rotate'.format(output), 'normal')
```
Comment (tobes) [inline py3status/modules/xrandr.py]: it is probably worth checking that the value or rotation is legal eg in  ['normal', 'inverted', 'left', 'right']

**Comment by ultrabug** [comment]
Hi @coxley , thank you for your contribution :)

I wanted to understand the motivation in adding this in the xrandr module rather than using the **xrandr_rotate** module please ?

**Comment by ultrabug** [comment]
Ok timeout @coxley with a lot of thank you. I'll implement the value validation then.

Cheers

**Comment by coxley** [comment]
@ultrabug: Crap, so sorry I let this fall-through. Didn't mean to be a drive-by contributor.

Yeah I forget exactly why I didn't use xrandr_rotate -- maybe it does the same thing but for some reason I thought it didn't meet my need. I see you merged it, but I'll try to play with it if I get some time.

**Comment by ultrabug** [comment]
@coxley np at all, thanks for contributing :)

**Pull request merged**